File: backend/Multibanca.Domain.Models/Multibanca/entregar_carpeta.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class entregar_carpeta : base_auditoria
    {
        public int id_entregar_carpeta { get; set; }
        public long id_expediente { get; set; }
        public bool? enviar_a_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/entregar_carpeta/models/entregar_carpeta.ts
export interface EntregarCarpeta {
  id_entregar_carpeta: number;
  id_expediente: number;
  enviar_a_reparo: boolean;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}
